src/test/java/com/telefonica/iot/cygnus/sinks/OrionSTHSinkTest.java

```diff
@@ -68,34 +69,79 @@
     private final String attrValue = "112.9";
     private final String attrMd = "[]";
     private final String serviceHeader = "vehicles";
-    private final String servicePathHeader = "4wheels";
-    private final String destination = "4wheelsCars";
+    private final String singleServicePathHeader = "4wheels";
+    private final String multipleServicePathHeader = "4wheels,4wheels";
```

Comment:
It is done that way ;) Have a look on https://github.com/telefonicaid/fiware-cygnus/pull/418/files#diff-a9addb8087c854151b0c0edb3a51e347R180